lib/app_theme.dart

```diff
@@ -23,19 +23,20 @@ class AppTheme {
   static const Color blackBackgroundColor = Colors.black;
   static const Color spaceBackgroundColor = Color(0xff242933);
 
-  static ThemeData lightTheme(Color color) =>
-      _buildTheme(color, backgroundColor: lightBackgroundColor);
+  static ThemeData lightTheme(Color color, {required bool useGestures}) =>
```

Comment:
Following the previous proposal, my preferred signature for the theming methods would be:

```suggestion
  static ThemeData lightTheme(WidgetRef ref, Color color) =>
```